Tweak icon component (#34)
hashicorp/flight: ember-flight-icons/README.md

# ember-flight-icons

An Ember addon for the icons from `flight`. Currently in development.

Goals:

* Accessible
* Easy to use
* Easy to maintain
* Sensible defaults

## Compatibility

* Ember.js v3.20 or above
* Ember CLI v3.20 or above
* Node.js v10 or above

## Installation

This package has not yet been published. Once it has, you can install it like any other Ember addon:

```bash
ember install @hashicorp/ember-flight-icons
```

## Usage

### Required usage

> The `name` must be specified. A default icon size of 16 will be provided.

Reviewer: ```suggestion
The `name` must be specified. A default icon size of 24 will be provided.
```
24x24 is the default size.

Author: oh gotcha, i assumed it was the other way around. any reason why? curious if more icons are 16 vs 24 rn. but, let's address that separately, i'll make a triage issue

```hbs
<FlightIcon @name="activity" />
```

### Optional usage

> Icons are set to a `viewBox` square size of 16 by default. Right, now, we support a `viewBox` square size of 16 or 24. Pass in `size` to use the larger 24.

Reviewer: ```suggestion
Icons are set to a `viewBox` square size of 24 by default. Right, now, we support a `viewBox` square size of 16 or 24. Pass in `size` to use the smaller 16 size.
```

```hbs

> <FlightIcon @name="activity" @size="24" />

Reviewer: ```suggestion
<FlightIcon @name="activity" @size="16" />
```

* easy to maintain
* sensible defaults

## Compatibility

* Ember.js v3.20 or above
* Ember CLI v3.20 or above
* Node.js v10 or above

## Installation

This package has not yet been published. Once it has, you can install it like any other Ember addon:

```bash
ember install @hashicorp/ember-flight-icons
```

## Usage

### Required usage

The `name` must be specified.

```hbs
<FlightIcon @name="activity" />
```

> The `size` must be specified. Set the icon size by the number, e.g. 16 or 24.

Author: melanie is ok with this

Icons are set to `fill="currentColor"` by default. To change this, set the desired color:

```hbs

> <FlightIcon @name="activity" @size="24" @color="red" />

Reviewer: ```suggestion
<FlightIcon @name="activity" @color="red" />
```

```

### Note

> The `id` is generated by the component to be unique.

Reviewer: ```suggestion
The `id` is automatically generated by the component, and will be unique.
```
